vault/changelog.release.2022-01-04.md

```diff
@@ -0,0 +1,106 @@
+---
+id: MQpi87qLnzKK5iK3FlxUp
+title: 0.76
+desc: ''
+updated: 1641320737273
+created: 1641314638791
+---
+
+Dendron 0.76 has sprouted  🌱
+
+We're adding some super powers to your templates with [[template variables|dendron.topic.templates#template-variables]], starting with dates. Do you find yourself updating date values in the content of your inserted templates? Now that can be taken care of automatically:
```

Comment:
this makes it sound like we're adding more things soon. would cut the `starting with dates`

Reply:
`Do you find yourself updating date values in the content of your inserted templates? Now that can be taken care of automatically:`

It's weird to have this after the intro of the feature. rephrase

```
Do you find yourself updating date values in the content of your inserted templates?  That can now be inserted automatically with [[template variables]]